pkg/helper/machinepools/helpers.go

```diff
@@ -159,6 +164,24 @@ func ParseTaints(taints string) ([]*cmv1.TaintBuilder, error) {
 	return taintBuilders, nil
 }
 
+func validateMachinePoolTaintEffect(taint string) error {
+	var allowedTaintEffects = []string{
```

Comment:
@den-rgb This can be package-scoped and not instantiated each time we call the function.